dump/dump_watch.cpp

```diff
@@ -42,6 +42,15 @@ void DumpWatch::interfaceAdded(sdbusplus::message::message& msg)
         sdbusplus::message::object_path objPath;
         DBusInteracesMap interfaces;
         msg.read(objPath, interfaces);
+        DBusInteracesMap::iterator iter = interfaces.begin();
+        if (interfaces.find("com.ibm.Dump.Entry.Hostboot") ==
+                interfaces.end() &&
+            interfaces.find("com.ibm.Dump.Entry.Hardware") ==
+                interfaces.end() &&
+            interfaces.find("com.ibm.Dump.Entry.SBE") == interfaces.end() &&
+            interfaces.find("xyz.openbmc_project.Dump.Entry.BMC") ==
+                interfaces.end())
+            return;
```

Comment:
if you remove in offload_manager.cpp it will not come here, no need special handling here, also you are not removing BMC in offload_manager but including here.

Reply:
Entry interface for sbe, hb, hw dumps is same as system dump. So, to exclude system dump i have added a check here.